Dockerfile

```diff
@@ -9,10 +13,13 @@ ENV NODE_VERSION ${NODE_VERSION:-9.3.0}
 ARG NPM_VERSION
 ENV NPM_VERSION ${NPM_VERSION:-5.6.0}
 
-# Set dependencies
-RUN yum clean all && yum -y install wget
+# Add the os specific default arguments
+ARG OS_IMG_NAME=centos
+ARG OS_IMG_VERSION=7
+
+# Add the os_wrapper script
+ADD os_wrapper.sh ./os_wrapper.sh
+ADD setup_env ./setup_env
```

Comment:
Does this work?
Shouldn't it be setup_env.sh ?

Reply:
I am adding the entire folder. setup_env is the folder containing the scripts.

Reply:
aah yes I thought this was Dockerfile.app in which case moving this source would have been required.
Yes now the one in line 22 suffices